regression/cbmc-cover/location-multiline-statement/dereference.h

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
:pick: Could use a comment explaining what this is here

Reply:
It is already explained in a comment of the `.desc` file.